packages/components/src/Toast.tsx

```diff
@@ -57,197 +63,149 @@ interface RegionVariants extends VariantProps<typeof region> {}
 interface IconVariants extends VariantProps<typeof icon> {}
 interface ToastVariants extends VariantProps<typeof toast> {}
 
-interface ToastContent {
-	children: ReactNode;
-}
-
-interface ToastValue extends IconVariants, ToastContent {}
-
-interface SnackbarContent {
+interface ToastValue {
 	title?: ReactNode;
-	description: ReactNode;
-	// biome-ignore lint/suspicious/noExplicitAny: <explanation>
-	action?: ReactElement<any>;
+	description?: ReactNode;
+	action?: ReactNode;
 }
 
-interface SnackbarValue extends IconVariants, SnackbarContent {}
-
-interface ToastOptions extends Omit<AriaToastOptions, 'timeout'> {}
+interface LPToastContent extends ToastValue, IconVariants {}
 
-interface ToastRegionProps<T> extends AriaToastRegionProps, RegionVariants {
-	state: ToastState<T>;
+interface ToastProps<LPToastContent> extends AriaToastProps<LPToastContent>, ToastVariants {
+	ref?: Ref<HTMLDivElement>;
 }
 
-interface ToastProps<T> extends AriaToastProps<T>, ToastVariants {
-	state: ToastState<T>;
-}
-
-const ToastRegion = <T extends ToastValue | SnackbarValue>({
-	state,
-	placement = 'bottom',
-	...props
-}: ToastRegionProps<T>) => {
-	const ref = useRef(null);
-	const { regionProps } = useToastRegion(props, state, ref);
-
-	return (
-		<div {...regionProps} ref={ref} className={region({ placement })}>
-			{state.visibleToasts.map((toast) => (
-				<Toast
-					key={toast.key}
-					toast={toast}
-					state={state}
-					variant={placement === 'bottom' ? 'default' : 'snackbar'}
-				/>
-			))}
-		</div>
-	);
+interface ToastRegionProps<LPToastContent>
+	extends Partial<AriaToastRegionProps<LPToastContent>>,
+		RegionVariants {}
+
+const animate = (fn: () => void) => {
+	if ('startViewTransition' in document) {
+		document
+			.startViewTransition(() => {
+				flushSync(fn);
+			})
+			.ready.catch(() => {});
```

Comment:
do we need this line?

Reply:
Figured I'd match how the RAC folks implemented it https://github.com/adobe/react-spectrum/blob/main/packages/%40react-spectrum/toast/src/ToastContainer.tsx#L46